doc/method/method-intro.md

```diff
@@ -87,35 +76,40 @@ The combination S + S' is formed in two steps:
  1. A mapping or _alignment_ step that identifies all
     nodes in S' that can be equated with nodes in S. There will often be nodes
     in S' that cannot be aligned to S.
- 2. A _merge_ step that creates the union, U = S + S', by adding to S the unaligned
-    taxa from S'. The attachment position of unaligned nodes from step 1 is
+ 2. A _merge_ step that creates the combination U = S + S', by adding to S the unaligned
+    taxa from S'. The attachment position of unaligned nodes from step 1
     is determined from nearby aligned nodes, either as a _graft_
     or an _insertion_. Examples of these two cases are given in
     [point to figure(s)].
 
 As a simple example, consider a genus represented in both
 taxonomies, but containing different species in the two:
 
-> [this would be a figure]   S = (c,d,e)a,  S' = (d,e,f)a
+> [this would be a figure]   S = (b,c,d)a,  S' = (c,d,e)a
 
-S and S' each have four nodes.  Suppose d, e, and a in S' are aligned
-to d, e, and a in S.  The only unaligned node is f, which is a
-sibling of d and e and therefore grafted as a child of a.  After the merge
+S and S' each have four nodes.  Suppose c, d, and a in S' are aligned
+to c, d, and a in S.  The only unaligned node is e, which is a
+sibling of c and d and therefore grafted as a child of a.  After the merge
 step, we have:
 
-> [figure] S + S' = (c,d,e,f)a
+> [figure] S + S' = (b,c,d,e)a
 
 One might call this merge heuristic 'my sibling's sibling is my
 sibling' or 'transitivity of siblinghood'.
 
-[what happened to b?]
+This is a very common pattern.  For example: in NCBI taxonomy, take a
+= _Bufo_, b = _B. spinosis_, c = _B. bufo_, d = _B. japonicus_, and
+in GBIF take a = _Bufo_, c = _B. bufo_, d = _B. japonicus_, e =
+_B. luchunnicus_.  There is no _B. spinosis_ in GBIF and no
+_B. luchunnicus_ in NCBI.  The _Bufo_ in the combined taxonomy has as
+its children copies of species records from both sources.  There are
+probably hundreds of thousands of similar simple grafting events (hard
+to count).
 
-[Real example: a = Bufo, c,d,e = Bufo bufo / spinosis / japonicus
-(etc.) in NCBI, d,e,f = Bufo bufo / japonicus / luchunnicus (etc.) in
-GBIF.  There are probably hundreds of thousands of other simple
-grafting events (hard to count).  I think the figures would be harder
-to read with real taxon names compared with schematic names (letters),
-but will take advice.]
+[In the presentation we could forego the schematic letters a, b,
+etc. in favor of the concrete _Bufo_ example.  I think the figures would be
+harder to read with real taxon names compared with schematic names
+(letters), but will take advice.]
```

Comment:
I think using the generic letters in the text is good, pointing to a figure with an example using real taxon names.